### PR TITLE
D2IQ-72776: Fix Kubernetes version in Kommander tutorial

### PR DESCRIPTION
**Pull request by ANeumann82**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-72776

## Description of changes being made
Fixed the Kubernetes Version of the created KonvoyCluster

## Checklist
- [X] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [X] Test all commands and procedures, if applicable.
- [X] Create your PR against `staging`, not `master`. 

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

**Review by tillt — APPROVED** [review]

File: pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ metadata:
 spec:
   cluster:
     kubernetes:
-      version: 1.26.4
+      version: 1.18.0
```
Comment (hectorj2f) [inline pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md:152]: 1.18.10 would be more exact

Comment (hectorj2f) [reply pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md:152]: That would be the default k8s 1.18 version that would be deployed with konvoy 1.6

**Review by makkes — APPROVED** [review]

File: pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md (hunk 1 above)
Comment (ANeumann82) [reply pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md:152]: Thanks, sounds good. I'll have to retry it with that version number

**Comment by ANeumann82** [comment]
I've updated the Min/Max/Default values on the Release Notes page as well, based on the return values from the webhook and  Hectors comment.

File: pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md (hunk 1 above)
Comment (ANeumann82) [reply pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md]: Works and looks good 👍

**Review by hectorj2f — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request merged**